src/mqtt.service.ts

```diff
@@ -215,7 +215,7 @@ export class MqttService {
       const f = fs.pop();
       const t = ts.pop();
       switch (f) {
-        // In case the filter level is '#', this is a match not matter whether
+        // In case the filter level is '#', this match always whether
```

Comment:
I think "[…] this match **always** whether […]" doesn't work here. It should rather be "[…] this is a match **no** matter whether the topic […]", or am I wrong?